src/main/webapp/styles/common.css

```diff
@@ -26,7 +26,21 @@ h3 {
   color: white;
   font-family: 'Questrial', sans-serif;
   font-size: 28px;
-  margin: 5px;
+  margin: 10px;
+}
+
+h4 {
+  color: white;
+  font-family: 'Questrial', sans-serif;
+  font-size: 22px;
```

Comment:
nit: it seems a little odd to have two different headers with the same formatting (e.g. same font size). If they need to be different headers, then they should probably be different in format.